src/crypto.rs

```diff
@@ -18,6 +18,12 @@ pub enum Algorithm {
     /// HMAC using SHA-512
     HS512,
 
+    /// ECDSA using SHA-256
+    ES256,
+
+    /// ECDSA using SHA-384
+    ES384,
```

Comment:
ring doesn't have ES512?

Reply:
It doesn't.
https://github.com/briansmith/ring/blob/master/src/ec.rs#L37-L41